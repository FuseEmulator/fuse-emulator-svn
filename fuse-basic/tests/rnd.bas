10 RANDOMIZE 1
20 PRINT RND, RND
30 RANDOMIZE 969631
40 PRINT RND, RND
