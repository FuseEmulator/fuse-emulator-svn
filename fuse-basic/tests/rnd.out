0.840188	0.394383
0.559937	0.00633111
0 OK
